src/domains/citizen-auth-app/env/weu-prod01/terraform.tfvars

```diff
@@ -1,14 +1,15 @@
-prefix                   = "io"
-env_short                = "p"
-env                      = "prod"
-domain                   = "citizen-auth"
-location                 = "westeurope"
-location_short           = "weu"
-location_string          = "West Europe"
-session_manager_location = "italynorth"
-instance                 = "prod01"
-lollipop_enabled         = true
-fastlogin_enabled        = true
+prefix                         = "io"
+env_short                      = "p"
+env                            = "prod"
+domain                         = "citizen-auth"
+location                       = "westeurope"
+location_short                 = "weu"
+location_string                = "West Europe"
+session_manager_location       = "italynorth"
+session_manager_location_short = "itn"
```

Comment:
The removal of this misleading config will happen with the removal of the OLD session manager?